operations/mimir-mixin/docs/playbooks.md

```diff
@@ -529,13 +529,13 @@ How to **investigate**:
 1. Pick a block and note its ID (`block` field in log entry) and tenant ID (`org_id` in log entry)
 1. Find the bucket used by the Mimir cell, such as checking the configured `blocks_storage_bucket_name` if you are using Jsonnet.
 1. Find out which Mimir component operated on the block last (e.g. uploaded by ingester/compactor, or deleted by compactor)
-   1. Determine when the partial block was created: `gsutil ls -l gs://${BUCKET}/${TENANT_ID}/${BLOCK_ID}`
+   1. Determine when the partial block was uploaded: `gsutil ls -l gs://${BUCKET}/${TENANT_ID}/${BLOCK_ID}`. Alternatively you can use `ulidtime` command from Mimir tools directory `ulidtime ${BLOCK_ID}` to find block creation time.
```

Comment:
Please ping me for doc reviews.

Reply:
👍 It's playbook. I don't see any changes from the docs team in this file, but will ping you next time. My bad.